setup.cfg

```diff
@@ -4,12 +4,12 @@ description_file  =  README.md
 summary = nxt_gem: A Python module for Graph Embedding Methods
 author = Jan Ernsting
 author_email = j.ernsting@uni-muenster.de
-url = https://github.com/jernsting/nxt_gem
+url = https://github.com/brunocarlin/nxt_gem
```

Comment:
You can change those links for your repo, but changing them in upstream would refer users of this package to your documentation. I would merge once the links are pointing to my repo again :-)